cmd/server/cmd/server.go

```diff
@@ -308,6 +323,9 @@ func runServer(sa *serverArgs, tmplRepo template.Repository, printf, fatalf shar
 	grpcOptions = append(grpcOptions, grpc.UnaryInterceptor(grpc_middleware.ChainUnaryServer(interceptors...)))
 
 	configManager.Register(adapterMgr)
+	if sa.useIL {
+		configManager.Register(ilEval)
```

Comment:
Yeah, I didn't expect you to take action. I don't recall why it needs eval, but IIRC it's to look at selectors? Unclear, and seems unnecessary. We should look into removing that dependency.